public/components/feature-switches/feature-switches.js

```diff
@@ -40,8 +42,11 @@ class FeatureSwitches {
 }
 
 function wfFeatureSwitchesController ($scope, wfPreferencesService) {
-    const featureSwitchKeys = [];
-    $scope.readableNames = {}
+    const featureSwitchKeys = [timelineFeatureSwitchKey, miniProfilesFeatureSwitchKey];
+    $scope.readableNames = {
+        [timelineFeatureSwitchKey]: 'Timeline',
+        [miniProfilesFeatureSwitchKey]: 'Mini Profiles',
```

Comment:
Sorry to nitpik but this should be `Mini profiles` (lower case p) throughout!

https://chat.google.com/room/AAAAw-4p-YQ/GgdcMpW-iUs/GgdcMpW-iUs?cls=10

Reply:
Fair one, thought I'd gotten it all correct this time!